tests/requirements-py3.txt

```diff
@@ -0,0 +1,4 @@
+pytest>=2.6.0
+pytest-twisted
+testfixtures
+jmespath
```

Comment:
Sorry, I thought the question wasn't pointed at me.
I don't have the answer -- I just followed the same order as requirements.txt,
it's in the `tests/requirements.txt` file for python2, too.

```
$ git grep jmespath
docs/topics/loaders.rst:    Requires jmespath (https://github.com/jmespath/jmespath.py) to run.
scrapy/loader/processors.py:        Query the input string for the jmespath (given at instantiation),
scrapy/loader/processors.py:        Requires : jmespath(https://github.com/jmespath/jmespath)
scrapy/loader/processors.py:        import jmespath
scrapy/loader/processors.py:        self.compiled_path = jmespath.compile(self.json_path)
scrapy/loader/processors.py:        """Query value for the jmespath query and return answer
scrapy/loader/processors.py:        :return: Element extracted according to jmespath query
tests/requirements-py3.txt:jmespath
tests/requirements.txt:jmespath
```
Seeing as it's not really important (only used in some itemloader proc), I'd prefer to not pull it in as a global installation requirement.